tips/TIP-0018/tip-0018.md

```diff
@@ -0,0 +1,3755 @@
+                                <td>uint8</td>
+                                <td>
+                                    Set to <strong>value 2</strong> to denote a <i>Metadata Block</i>.
+                                </td>
+                            </tr>
+                            <tr>
+                                <td>Data Length</td>
+                                <td>uint32</td>
+                                <td>
+                                    Length of the following data field in bytes.
+                                </td>
+                            </tr>
+                            <tr>
+                                <td>Data</td>
+                                <td>ByteArray</td>
+                                <td>Binary data.</td>
+                            </tr>
+                        </table>
+                    </details>
+                    <details>
+                        <summary>Tag Block</summary>
+                        <blockquote>
+                            Defines an indexation tag to which the output can be indexed by additional node plugins.
+                        </blockquote>
+                        <table>
+                            <tr>
+                                <td><b>Name</b></td>
+                                <td><b>Type</b></td>
+                                <td><b>Description</b></td>
+                            </tr>
+                            <tr>
+                                <td>Block Type</td>
+                                <td>uint8</td>
+                                <td>
+                                    Set to <strong>value 3</strong> to denote a <i>Tag Block</i>.
+                                </td>
+                            </tr>
+                            <tr>
+                                <td>Tag Length</td>
+                                <td>uint8</td>
+                                <td>
+                                    Length of the following tag field in bytes.
+                                </td>
+                            </tr>
+                            <tr>
+                                <td>Tag</td>
+                                <td>ByteArray</td>
+                                <td>Binary indexation data.</td>
+                            </tr>
+                        </table>
+                    </details>
+                </td>
+            </tr>
+        </table>
+    </details>
+</table>
+
+### Additional Transaction Syntactic Validation Rules
+
+- `Amount` field must fulfill the dust protection requirements.
+- `Native Tokens Count` must not be greater than `Max Native Tokens Count`.
+- `Native Tokens` must be lexicographically sorted based on `Token ID`.
+- Each <i>Native Token</i> must be unique in the set of `Native Tokens` based on its `Token ID`. No duplicates are
+  allowed.
+- It must hold true that `0` ≤ `Unlock Conditions Count` ≤ `4`.
```

Comment:
This condition is incorrect as there must always be an `AddressUnlockCondition`

Reply:
Good catch!